luigi_pipeline/seqr_loading.py

```diff
@@ -205,6 +208,7 @@ class SeqrMTToESTask(HailElasticSearchTask):
     vep_runner = luigi.ChoiceParameter(choices=['VEP', 'DUMMY'], default='VEP', description='Choice of which vep runner to annotate vep.')
 
     reference_ht_path = luigi.Parameter(default=None, description='Path to the Hail table storing the reference variants.')
+    interval_ref_ht_path = luigi.Parameter(description='Path to the Hail Table storing interval-keyed reference data.')
```

Comment:
all the other params have `default=None`, you probably want to add that here too